bin/convert-argv.js

```diff
@@ -553,5 +554,11 @@ module.exports = function(optimist, argv, convertOptions) {
 			console.error("Use --help to display the CLI options.");
 			process.exit(-1); // eslint-disable-line
 		}
+		var webpackOptionsValidationErrors = validateWebpackOptions(options);
+
+		if (webpackOptionsValidationErrors.length) {
+			console.error("Configuration is invalid. Fix the following errors:", webpackOptionsValidationErrors);
+			process.exit(-1); // eslint-disable-line
+		}
```

Comment:
remove this block and catch the exceptions thrown by the API in `bin/webpack.js` instead.